calculatorUICOOP/Views/MainPage.xaml.cs

```diff
@@ -13,58 +9,65 @@ namespace calculatorUICOOP
     public partial class MainPage : ContentPage
     {
         #region Fields and Properties
```

Comment:
You have 2 "Fields and Properties" regions. Maybe consolidate them, and then separate the fields from the properties.